Armadillo/src/main/java/com/scribd/armadillo/playback/error/ArmadilloHttpErrorHandlingPolicy.kt

```diff
@@ -0,0 +1,36 @@
+package com.scribd.armadillo.playback.error
+
+import com.google.android.exoplayer2.C
+import com.google.android.exoplayer2.ParserException
+import com.google.android.exoplayer2.upstream.DefaultLoadErrorHandlingPolicy
+import com.google.android.exoplayer2.upstream.LoadErrorHandlingPolicy
+import java.net.SocketTimeoutException
+import java.net.UnknownHostException
+
+class ArmadilloHttpErrorHandlingPolicy : DefaultLoadErrorHandlingPolicy(DEFAULT_MIN_LOADABLE_RETRY_COUNT) {
+    override fun getRetryDelayMsFor(loadErrorInfo: LoadErrorHandlingPolicy.LoadErrorInfo): Long {
+        return if (loadErrorInfo.exception is UnknownHostException || loadErrorInfo.exception is SocketTimeoutException) {
+            //retry every 10 seconds for potential loss of internet -keep buffering - internet may later succeed.
+            if (loadErrorInfo.errorCount > 8) {
+                C.TIME_UNSET //stop retrying after a few minutes
+            } else {
+                //exponential backoff based on a 10 second interval
```

Comment:
Can you put in what the max backoff will be? My reading is that will be up to 40 minutes, but it should be written down.